applications/nrf_desktop/src/events/click_event.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#ifndef _CLICK_EVENT_H_
+#define _CLICK_EVENT_H_
+
+/**
+ * @brief Click Event
+ * @defgroup click_event Click Event
+ * @{
+ */
+
+#include "event_manager.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** @brief Click types list. */
+#define CLICK_LIST		\
+	X(NONE)			\
+	X(SHORT)		\
+	X(LONG)			\
+	X(DOUBLE)		\
+	X(HOLD_ON_INIT)		\
```

Comment:
I prefer to remove `HOLD_ON_INIT`.

We could change the logic to check if long click happens within specific time after init (if so perform required action in the module that does it).

Reply:
Ok, I will move this part then